docs/source/models/causes/malaria/index.rst

```diff
@@ -0,0 +1,289 @@
+Malaria is modeled separately for inside versus outside sub-Saharan Africa. 
+However, Ethiopia is modeled as outside sub-Saharan Africa since it exhibits 
+epidemiological trends and has data availability and quality more akin to non-African 
+settings. 
+
+For countries in sub-Saharan Africa (Nigeria in the nutrition optimization model) 
+the PfPR surveys by the Malaria Atlas Project (MAP) were used to predict malaria 
+rates for both the non-fatal and fatal models. 
+
+For countries outside of sub-Saharan Africa (Ethiopia and Pakistan in this project) surveillance 
+systems tend to be stronger and so national and subnational case reports are the 
+primary data source for the nonfatal model. 
+
+Nonfatal modeling in all countries combined data sources to generate pixel-level predictions 
+of clinical incidence rates. These predications were combined with high-resolution gridded population 
+data to estimate total cases per pixel-year. These were then aggregated to GBD 
+national/ subnational areas. 
+
+For the fatal model, MAP data was used to estimate the cause specific mortality rate. 
+This was then combined with the incidence rate from the nonfatal model in order to 
+find estimates for case fatality rate. This in turn was used to find annual mortality 
+rates for each location. [GBD-2019-Capstone-Appendix-Malaria-2020]_
+
+.. todo::
+
+   GBD 2021 includes COVID shocks. We need to decide if and how we want to include this information. 
+
+A systematic review of malaria severity was conducted, from which simple 
+severity splits were obtained and applied across all cases:
+
+
+.. list-table:: Malaria severity splits
+	:widths: 5 50 5
+	:header-rows: 1
+	
+	* - Severity level
+	  - Lay description
+	  - Disability weight
+	* - Mild
+	  - Mild malaria	Has a low fever and mild discomfort but no difficulty with daily activities.	
+	  - 0.006 (0.002–0.012)
+	* - Moderate
+	  - Has a fever and aches and feels weak, which causes some difficulty with daily activities. 
+	  - 0.051 (0.032–0.074)
+	* - Severe
+	  - Has a high fever and pain and feels very weak, which causes great difficulty with daily activities. 
+	  - 0.133 (0.088–0.19)
+
+
+GBD Hierarchy
+-------------
+
+.. image:: malaria_cause_hierarchy.svg
+
+Cause Model Diagram
+-------------------
+
+.. image:: malaria_cause_model.svg
+
+
+S: **S**\ usceptible to malaria
+
+I: **I**\ nfected and currently experiencing malaria
+
+
+Model Assumptions and Limitations
+---------------------------------
+
+Malaria has been modeled extensively by other teams. For this model, 
+we will not be including any information on vectors and so it is not 
+appropriate for interventions targetting malaria prevention or treatment 
+directly. 
+
+There is evidence that people living in malaria endemic areas do gain immunity over 
+their lifetimes. We assume this is represented in prevalence and incidence rates from 
+GBD. We do not include in this model any gains in malaria resistance from prior exposure. 
+
```

Comment:
I think this is great to note! For our purposes, we really just care about malaria DALYs attributable to CGF since our model is mainly about CGF and not about malaria transmission dynamics - maybe this would be good additional context to note here about why we don't need to get into any of these additional details.